src/datalink/mod.rs

```diff
@@ -229,6 +229,15 @@ impl NetworkInterface {
     }
 }
 
+/// Represents an IP address and subnet mask
+#[derive(Clone, PartialEq, Eq, Debug, Hash)]
+pub struct IpNetmask {
+    /// IP address
+    pub ip: IpAddr,
+    /// Subnet mask
+    pub netmask: Option<IpAddr>,
```

Comment:
Actually, it says that `ifa_addr` could also be null, so now I'm not sure what to think!